setup.py

```diff
@@ -40,7 +40,8 @@
         'pyrsistent==0.15.4',
         'jams>=0.3.2',
         'numpy>=1.13.3',
-        "soxbindings>=1.2.0;platform_system!='Windows'",
+        "soxbindings>=1.2.2;platform_system!='Windows'",
+        'pyloudnorm',
```

Comment:
pin to specific version?

Reply:
I think SoxBindings will work going into the future. After all, I maintain it. But we can pin it if you want.

Reply:
oh I was referring more to pyloudnorm, but I guess this applies to both

Reply:
following offline discussion we decided NOT to pin